src/cogs/trivia.py

```diff
@@ -49,6 +49,46 @@ def tally_scores(
                 (loser, ))
         self.db.commit()
 
+    @commands.command(usage="#somechannel")
+    @commands.has_permissions(manage_channels=True)
+    async def tchan(self, ctx):
+        """
+        Set channels designated for the trivia game
+        """
+        values = self.bot.getConfig('Trivia', 'channels')
+
+        channel = ctx.message.channel_mentions[0].id
+
+        if channel in values:
+            return await ctx.send('Channel Already Set!')
+
+        values.append(str(channel))
+
+        self.bot.setConfig('Trivia', 'channels', values)
+
+        await ctx.send("Channel set!")
+
+    @commands.command(usage="#somechannel")
+    @commands.has_permissions(manage_channels=True)
+    async def rmtchan(self, ctx):
```

Comment:
During testing, this command always returns "Channel not set!" even when the channel is, in fact, set.